doc/Pacemaker_Explained/en-US/Ch-Resources.txt

```diff
@@ -611,6 +611,8 @@ you need to add a +monitor+ operation to the resource's definition.
 |interval
 |0
 |How frequently (in seconds) to perform the operation. A value of 0 means never.
+ Otherwise, we talk about a 'recurring action', which is typical especially for
+ <<s-resource-monitoring,monitor>>.
```

Comment:
I'd prefer:

A positive value defines a 'recurring action', which is typically used with <<s-resource-monitoring,monitor>> operations.